app/javascript/react/screens/App/Overview/screens/MappingWizard/helpers.js

```diff
@@ -21,16 +22,17 @@ export const createTransformationMappings = (
     []
   );
 
-  const datastoresUrlRegEx = /\/api\/data_stores\/\d{1,}/;
+  const sourceStoragesUrlRegEx = /\/api\/data_stores\/\d{1,}/;
+  const targetStoragesUrlRegEx = /\/api\/data_stores|cloud_volumes\/\d{1,}/;
```

Comment:
Ok real quick, this could be crazy, but what about using the constants from  `MappingWizardConstants.js` here?  (only for the `cloud_volumes`/`cloud_networks') part

Reply:
Yes, that is a great suggestion.  It seems likely that a least the OSP storages urls/things will be changing, and the whole point of those constants is to not have to chase down every `cloud_volumes`

Good call